compatibility/sandbox-migration/util.bzl

```diff
@@ -3,7 +3,7 @@
 
 load("@os_info//:os_info.bzl", "is_linux")
 
-def migration_test(name, versions, tags, **kwargs):
+def migration_test(name, versions, tags, quick_tags, **kwargs):
```

Comment:
did you mean to use `quick_tags` below?

Reply:
:facepalm: 99da8660823bee3ad3d27d5b4cc8e6e3380da1d0